operations/network/management_network/replace_switch.md

```diff
@@ -1,16 +1,25 @@
 # Replace Switch
 
-### Do not plug in a switch that is not configured.
-- This can cause unpredicted behavior and network outages. 
-#### Prerequisites 
+## Do not plug in a switch that is not configured.
```

Comment:
To be consistent with the rest of the docs:

**CAUTION:** Do not plug in a switch that is not configured. This can cause unpredictable behavior and network outages.